lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp

```diff
@@ -114,6 +114,22 @@ class DecomposeAtenMatmulOp : public OpRewritePattern<AtenMatmulOp> {
 };
 } // namespace
 
+// Decompose torch.expand into torch.broadcast_to op.
+namespace {
+class DecomposeAtenExpandOp : public OpRewritePattern<AtenExpandOp> {
+public:
+  using OpRewritePattern::OpRewritePattern;
+  LogicalResult matchAndRewrite(AtenExpandOp op,
+                                PatternRewriter &rewriter) const override {
+    Value input = op.self();
+    Value size = op.size();
+    rewriter.replaceOpWithNewOp<AtenBroadcastToOp>(op, op.getType(), input,
```

Comment:
We can assert the `op.implicit` is false here. Just to be sure, can you double if the IR for e2e test does have the `implicit` to as `false`?